packages/AddressSearchMap/AddressSearchMap.vue

```diff
@@ -3,6 +3,7 @@
     <div class="myvic-address-search-map__map-container">
       <div class="myvic-address-search-map__address-search-container">
         <address-search
+          v-if="!ie11"
```

Comment:
Are we hiding only the search field and not other map components below?

Reply:
The map is already hidden in IE browser. The only component that is covering the ME link is the search field.